### DISCUSSION TITLE
examples v7 -> 'Custom Input' demo is bad

### DISCUSSION THREAD

**Discussion by Young6-me** [react-hook-form/react-hook-form#7330] [Q&A]
Material UI - Input   and  Strap - Input is not work.

<img width="1212" alt="WeChata4022bf152deebac52d4fd57feb4e993" src="https://user-images.githubusercontent.com/22109906/146939265-194506e1-31a9-414c-bdcc-9cedbcd6117c.png">

I want to know if this is normal or there is a problem?

**Comment by bluebill1049** [comment]
can you share the link?

**Reply by Young6-me** [reply]
[https://codesandbox.io/s/react-hook-form-custom-input-hmyeu](https://codesandbox.io/s/react-hook-form-custom-input-hmyeu)

This is the official v7 example.
I tried to learn this example, but found ‘Material UI-Input’ and ‘Strap-Input’ is not work.

**Reply by bluebill1049** [reply]
which page did you get this link? i will get it updated.

**Reply by Young6-me** [reply]
[https://github.com/react-hook-form/react-hook-form/tree/master/examples](https://github.com/react-hook-form/react-hook-form/tree/master/examples)

![image](https://user-images.githubusercontent.com/22109906/147182259-b31c5cb9-b5de-4a56-bcf9-58c430b9522a.png)

**Reply by bluebill1049** [reply]
updated.
